oxd-server/doc/license.txt

```diff
@@ -0,0 +1,651 @@
+GNU Affero General Public License
```

Comment:
@duttarnab Do we really need this license file here?

Reply:
Removed this file